engine/dlib/src/dmsdk/dlib/transform.h

```diff
@@ -196,20 +207,6 @@ namespace dmTransform
         return dmVMath::Point3(rotate(t.GetRotation(), mulPerElem(dmVMath::Vector3(p), t.GetScale())) + t.GetTranslation());
     }
 
-    /*#
-     * Apply the transform on a point, but without scaling the Z-component of the point (includes the transform translation).
-     * @name ApplyNoScaleZ
-     * @param t [type: dmTransform::Transform&] Transform
-     * @param p [type: dmVMath::Point3&] Point
-     * @return point [type: dmVMath::Point3] Transformed point
-     */
-    inline dmVMath::Point3 ApplyNoScaleZ(const Transform& t, const dmVMath::Point3 p)
-    {
-        dmVMath::Vector3 s = t.GetScale();
-        s.setZ(1.0f);
-        return dmVMath::Point3(rotate(t.GetRotation(), mulPerElem(dmVMath::Vector3(p), s)) + t.GetTranslation());
-    }
```

Comment:
I think this is acceptable, as we already will designate the entire release as a breaking change.

We ofc need to update Spine + Rive externsions.